app/util/analytics.py

```diff
@@ -39,7 +40,7 @@ class AnalyticsCollector:
 
     def __init__(self, application_type):
         self.application_type = application_type
-        self.run_id = ""
+        self.run_id = uuid.uuid1().hex
```

Comment:
I think `uuid.uuid1()` will be enough. UUID is a very popular solution for ID in DB and it is used in default presentation.